### PR TITLE
Shutdown stability

### PR DESCRIPTION
**Pull request by mimi1vx**
https://progress.opensuse.org/issues/105506

Added clear command to tests using same console as shutdown to make needle match more stable

https://openqa.opensuse.org/tests/2181124#step/shutdown/1

### CODE REVIEW

File: tests/shutdown/shutdown.pm [hunk 1]
```diff
@@ -3,8 +3,9 @@ use base "openQAcoretest";
 use testapi;
 
 sub run {
-    wait_screen_change { send_key 'ctrl-alt-f3' };
-    type_string "poweroff\n";
+    send_key('ctrl-alt-f3', wait_scren_change => 1) ;
```
Comment (okurz) [inline tests/shutdown/shutdown.pm:6]: that still has a typo.

File: tests/shutdown/shutdown.pm [hunk 2]
```diff
@@ -3,8 +3,9 @@ use base "openQAcoretest";
 use testapi;
 
 sub run {
-    wait_screen_change { send_key 'ctrl-alt-f3' };
-    type_string "poweroff\n";
+    send_key('ctrl-alt-f3', wait_screen_change => 1) ;
```
Comment (Martchus) [inline tests/shutdown/shutdown.pm:6]: Is `wait_screen_change => 1` required if we're using an `assert_screen` in the next line anyways?

Comment (okurz) [reply tests/shutdown/shutdown.pm:6]: No, shouldn't be required. That was my point

File: lib/utils.pm [hunk 3]
```diff
@@ -6,7 +6,12 @@ use strict;
 use testapi;
 use File::Basename qw(basename);
 
-our @EXPORT = qw(switch_to_x11 wait_for_desktop ensure_unlocked_desktop wait_for_container_log);
+our @EXPORT = qw(clear_root_console switch_to_x11 wait_for_desktop ensure_unlocked_desktop wait_for_container_log);
+
+sub clear_root_console {
+    enter_cmd("clear");
+    assert_screen "root-console";
```
Comment (okurz) [inline lib/utils.pm:13]: please prefer single tickets as long as you don't want perl code to be evaluated. So better

```suggestion
    enter_cmd('clear');
    assert_screen 'root-console';
```

File: tests/install/test_distribution.pm [hunk 4]
```diff
@@ -11,9 +11,10 @@ sub run {
     save_screenshot;
     assert_script_run('zypper -n ref -f',                              60);
     assert_script_run('zypper -n in os-autoinst-distri-opensuse-deps', 600);
-    type_string "clear\n";
+    enter_cmd("clear");
+    assert_screen "root-console";
```
Comment (okurz) [inline tests/install/test_distribution.pm:15]:
```suggestion
    clear_root_console;
```

**Review by okurz — CHANGES_REQUESTED** [review]

**Review by Martchus — APPROVED** [review]

File: lib/utils.pm (hunk 3 above)
Comment (okurz) [reply lib/utils.pm]: seems you overlooked some other changes where there are still double ticks. Please check again (or I did something wrong?)

**Review by okurz — APPROVED** [review]
You still did introduce double quotes for strings that you don't want to expand variables in. Apparently I can not make myself clear. So fine, let's merge it anyways :)

**Pull request merged**